src/gluonts/mx/distribution/distribution.py

```diff
@@ -353,18 +354,17 @@ def _tensor_cdf_bisection(
 
     def quantile(self, level: Tensor) -> Tensor:
         r"""
-
         Calculates quantiles for the given levels.
 
         Parameters
         ----------
-        level
+        level:
             Level values to use for computing the quantiles.
             `level` should be a 1d tensor of level values between 0 and 1.
 
         Returns
         -------
-        quantiles
```

Comment:
Why is colon needed? We tried to follow the [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html) for docstrings, where colon is used to separate arg name and type. But type is redundant since it's already in the method signature, so I would prefer not to have it.

Reply:
yes, I think that : is standard and the type is optional, not that there is " because of type (which is not present here)